src/index.html

```diff
@@ -1,13 +1,64 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500; 700&display=swap"
+      rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
+      <div class="product-card__image">
+        <img
+          src="images/imac.jpeg"
+          class="image"
+          alt="iMac"
+        >
+      </div>
+
+      <div class="product-card__description description">
+        <p class="description__title">
+          Моноблок APPLE A1419 iMac 27&#34; Retina 5K (MNED2UA/A)
+        </p>
+        <p class="description__code">
+          Код товара: 195434
+        </p>
+        <div class="description__rating rating">
+          <div class="rating__rate-stars rate-stars">
+            <div class="rate-stars__star"></div>
+            <div class="rate-stars__star"></div>
+            <div class="rate-stars__star"></div>
+            <div class="rate-stars__star"></div>
+            <div class="rate-stars__star"></div>
+          </div>
+
+          <span class="rating__rate-rewiews">
+            Отзывов: 5
+          </span>
+        </div>
+
+        <div class="description__price">
+          <span class="price-text">
+            Цена:
+          </span>
+          <span class="price-value">
+            69 999 грн
+          </span>
+        </div>
+        <a
+          href="#"
+          class="description__buy"
```

Comment:
```suggestion
          class="product-card__buy-button"
```
It's more related to the card itself, not to description

Reply:
Когда я пытаюсь написать так линтер ругается - [скриншот](https://prnt.sc/111c7um)